src/sentry/models/relocation.py

```diff
@@ -81,6 +81,15 @@ class EmailKind(Enum):
         def get_choices(cls) -> list[tuple[int, str]]:
             return [(key.value, key.name) for key in cls]
 
+    class Provenance(Enum):
```

Comment:
If you inherit from IntEnum you won't have to use `.value` when using the enum.

Reply:
Done.